src/components/MainNav/MyProfile/MyProfile.js

```diff
@@ -47,6 +55,38 @@ export default class MyProfile extends Component {
     this.getProfileImage = this.getProfileImage.bind(this);
   }
 
+  componentDidMount() {
+    const { modules } = this.props;
+    const navLinks = modules.settings.map(({ links }) => links);
+
+    if (navLinks && Array.isArray(navLinks)) {
+      const links = navLinks[0].userDropdown;
```

Comment:
Why are we looking only at the first element of `navLinks`? (I'm not saying this is wrong, just not following the logic yet.)

Reply:
Yep, that should be changed to not only check the first module with the links object.

Reply:
Yep that should be changed to not only handle the first module with `links`, but every of them

Reply:
you are correct. changed